components/Quote/getQuote.ts

```diff
@@ -0,0 +1,5 @@
+import { QuoteResponse } from "../../pages/api/quote";
+
+export async function getQuote(): Promise<QuoteResponse> {
```

Comment:
```suggestion
export function getQuote(): Promise<QuoteResponse> {
```
unnecessary async keyword since a promise is returned